README.md

```diff
@@ -1,2 +1,3 @@
 # hawk.types
-Common Hawk types for all projects
+
+Types for Hawk's modules
```

Comment:
what modules? Pls describe all the list of them

Reply:
still no answer

Reply:
l8t